### PR TITLE
Add blocks to database and include lots more dummy data in `fixture.sql`

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
Part of #83

### CODE REVIEW

File: scripts/fixtures.sql [hunk 1]
```diff
@@ -21,8 +30,78 @@ begin
...
-call department('Computer Science');
-call department('Physics');
+create procedure create_departments()
+language plpgsql
+as $$
+declare
+    root realms.id%type;
+begin
+    insert into realms (name, parent, path_segment)
+        values ('Lectures', 0, 'lectures')
```
Comment (JulianKniephoff) [inline scripts/fixtures.sql:40]: `insert`-s are now formatted inconsistently. I tend to not indent the `values` clause, but also don't really have a hard opinion either way. I have **seen**

```sql
insert into ...
values ...
```

much more often than

```sql
insert into ...
    values ...
```

though.

File: scripts/fixtures.sql [hunk 2]
```diff
@@ -1,14 +1,23 @@
-create procedure department(name text)
+create procedure department(lectures_root realms.id%type, name text)
```
Comment (JulianKniephoff) [inline scripts/fixtures.sql:1]: I think I would reverse the order of all of these `procedure` declarations. :thinking:

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
Just some minor stylistic things.

File: scripts/fixtures.sql (hunk 1 above)
Comment (LukasKalbertodt) [reply scripts/fixtures.sql]: I went with the indented version as I find it a lot easier to read. And regarding what is used more in the world: I WONDER IF THERE IS ANOTHER THING MOST SQL CODE DOES THAT WE DO NOT AGREE WITH...

**Review by JulianKniephoff — APPROVED** [review]

**Pull request merged**